Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -18,19 +18,52 @@
 #import "ABContactInternal.h"
 #import "UWP/WindowsApplicationModelContacts.h"
 
+@interface __ABContactOperation : NSObject
```

Comment:
Well it seems like a lot of the classes are internal only here. The pattern I've been following has been that any private classes can go in the internal header for the file that needs to know of it.

---
In reply to: [76687605](https://github.com/Microsoft/WinObjC/pull/935#discussion_r76687605) [](ancestors = 76687605,76672379)